src/hooks/v3/useAllV3Routes.ts

```diff
@@ -1,10 +1,9 @@
 import { Currency } from '@uniswap/sdk-core';
-import { useActiveWeb3React } from 'hooks';
-import { Pool } from 'lib/src/pool';
-import { Route } from 'lib/src/route';
+import { Pool, Route } from 'v3lib/entities';
 import { useMemo } from 'react';
-// import { useUserSingleHopOnly } from '../state/user/hooks'
+import { useUserSingleHopOnly } from 'state/user/hooks';
```

Comment:
Yes, it was imported from wrong path, it should be 
`import { useUserSingleHopOnly } from 'state/user/v3/hooks';
`

Since I already added separate isSingleHop flag in userV3 state